diff --git a/node_modules/@oclif/core/lib/interfaces/parser.d.ts b/node_modules/@oclif/core/lib/interfaces/parser.d.ts
index b8fdcda..d54df1b 100644
--- a/node_modules/@oclif/core/lib/interfaces/parser.d.ts
+++ b/node_modules/@oclif/core/lib/interfaces/parser.d.ts
@@ -113,6 +113,7 @@ export declare type FlagProps = {
     required?: boolean;
     dependsOn?: string[];
     exclusive?: string[];
+    sensitive?: boolean;
 };
 export declare type BooleanFlagProps = FlagProps & {
     type: 'boolean';
